langchain/callbacks/tracers/base.py

```diff
@@ -322,3 +288,33 @@ def __deepcopy__(self, memo: dict) -> BaseTracer:
     def __copy__(self) -> BaseTracer:
         """Copy the tracer."""
         return self
+
+    def _on_llm_start(self, run: Run) -> None:
```

Comment:
@agola11 these are what you had in mind, right?

Reply:
yup exactly!